hudi-spark-datasource/hudi-spark-common/src/main/java/org/apache/hudi/internal/DataSourceInternalWriterHelper.java

```diff
@@ -83,8 +84,9 @@ public void onDataWriterCommit(String message) {
   public void commit(List<WriteStatus> writeStatuses) {
     try {
       List<HoodieWriteStat> writeStatList = writeStatuses.stream().map(WriteStatus::getStat).collect(Collectors.toList());
-      writeClient.commitStats(instantTime, writeStatList, Option.of(extraMetadata),
-          CommitUtils.getCommitActionType(operationType, metaClient.getTableType()));
+      writeClient.commitStats(instantTime, writeStatList, Collections.emptyList(), Option.of(extraMetadata),
+          CommitUtils.getCommitActionType(operationType, metaClient.getTableType()), Collections.emptyMap(), Option.empty(),
+          true, Option.empty());
```

Comment:
Note to Reviewer:
here is where we set `skipStreamingWritesToMetadataTable` to true.